srcpkgs/posixovl/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'posixovl'
+pkgname=posixovl
+version=1.3
+revision=1
+only_for_archs="aarch64 armv5te armv5tel armv6l arm6hf armv7l armv7hf i686 x86_64"
```

Comment:
I don't think this is necessary, what is the problem that requires excluding mips and *-musl ?

Reply:
Fails to build on all *-musl, erroring out with some conflicting types. Before getting sucked into upstream fixes, I figured that it wouldn't hurt go ahead and to push the working archs.